proto/storage/proto.lock

```diff
@@ -9888,6 +9892,10 @@
               {
                 "name": "CERTIFIED_RHEL_SCAN_UNAVAILABLE",
                 "integer": 3
+              },
+              {
+                "name": "CONTENT_SET_UNAVAILABLE"
+                "integer": 4
```

Comment:
This also needs to be removed to make the style-check happy

Reply:
I hope that this was the source of the style errors. I've been searching for them for a bit before reading your comment here